app/models/bank_account.rb

```diff
@@ -81,5 +81,14 @@ def grouped_by_currency(def_currency)
         currencies.index(ba.first)
       end
     end
+
+    def create_defaults(organization)
+      [*DEFAULT_VALUES[:bank_accounts]].each do |account|
+        BankAccount.find_or_create_by(
```

Comment:
@dissident it will be better to write `organization.bank_accounts.find_or_create(name: account)` so you will have an error if `organization` is nil. It's just small additional check that code will be worked as expected with wrong values.